packages/typespec-client-generator-core/test/packages/utils.ts

```diff
@@ -0,0 +1,54 @@
+import { strictEqual } from "assert";
+import {
```

Comment:
name the file to utils.test.ts?

Reply:
ok, I see, this is just a util for testing, not test on the utils, then the name should be fine.